### PR TITLE
[DON'T MERGE] chore: bump versions to Angular 7.0-rc.0

### PR DESCRIPTION
**Pull request by sis0k0**

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#sis0k0/ng-7

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#sis0k0/ng-7

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#sis0k0/ng-7

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#sis0k0/ng-7

**Comment by SvetoslavTsenov** [comment]
test branch_nativescript_dev_webpack#svetoslavtsenov/ng-7-tests

File: e2e/modal-navigation-ng/package.json [hunk 1]
```diff
@@ -13,25 +13,25 @@
...
+    "@angular/forms": "~7.0.0",
+    "@angular/http": "~7.0.0",
+    "@angular/platform-browser": "~7.0.0",
+    "@angular/platform-browser-dynamic": "~7.0.0",
+    "@angular/router": "~7.0.0",
     "nativescript-angular": "file:../../nativescript-angular",
     "nativescript-theme-core": "~1.0.4",
     "reflect-metadata": "~0.1.8",
-    "rxjs": "~6.1.0",
+    "rxjs": "~7.0.0",
```
Comment (elena-p) [inline e2e/modal-navigation-ng/package.json]: rxjs version should be 6.3.3

Comment (vchimev) [reply e2e/modal-navigation-ng/package.json]: Thanks, it is `6.3.3`: https://github.com/NativeScript/nativescript-angular/blob/master/nativescript-angular/package.json#L55.

**Comment by vchimev** [comment]
Merged with https://github.com/NativeScript/nativescript-angular/commit/37467097d1f7f91bb6d9ad0743846d61be91181c.

**Pull request closed without merging**